float32 lw_vel
float32 rw_vel
float32 bw_vel